doc/readme.md

```diff
@@ -0,0 +1,16 @@
+# Trapperkeeper Authorization
```

Comment:
Works for me. I can move any info here not in the readme to the readme and then maybe rename this file "index.md". That might help clarify the distinction.